dashboard-frontend/src/pages/NewStudyPage/controllers/simple.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  initialiseFieldState,
+  updateFieldState,
+  Event,
+  getFieldState,
+} from '../../../helpers/state';
+import { Conf } from '../../../types/conf';
+import { ConfBase, FieldState } from '../../../types/form';
+
+const simple = (
+  conf: ConfBase,
+  localFormData?: Conf,
+  event?: Event,
+  fieldState?: FieldState[]
+) => {
+  const type: string = 'confObject';
```

Comment:
Why is this needed? I thought it was the other way around. `confObject` just declares that it uses this controller, why does the controller know about the object type?

Reply:
No that's valid, there's no reason for it, it was just an extra guardrail in case any other conf type slipped in but I don't see why this would happen.